styles/modals-all.css

```diff
@@ -0,0 +1,101 @@
+.component-modal {
+  width: 90%;
+  height: 250px;
+  background: white;
+  justify-self: center;
+  border-radius: 15px;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.modal1 {
+  width: 90%;
+  height: 80%;
+  border: 1px solid rgb(226, 223, 225);
+  display: flex;
+  border-radius: 0.2rem;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.modal1-heading {
+  width: 100%;
+  height: 30%;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  border-bottom: 1px solid rgb(226, 223, 225);
+}
+
+.modal1-heading > h2 {
+  font-weight: 500;
+}
+
+.modal1-content {
+  width: 100%;
+  height: 40%;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  border-bottom: 1px solid rgb(226, 223, 225);
+}
+
+.modal1-footer > button {
+  padding: 0.5rem 1.5rem;
+  font-size: 1rem;
```

Comment:
feat: for colors, padding and margins could have used css variable to avoid repetition and have a consistent look and feel